TemplePlus/ui/ui_popup.cpp

```diff
@@ -3,27 +3,26 @@
 #include "ui.h"
 #include "tig/tig_keyboard.h"
 
-UiPopup uiPopup;
+UiPopupHandler uiPopupHandler;
```

Comment:
I think I've unified this with UiPopup in my branch

Reply:
Right.  I changed a few things around in addition to renaming.  I see a messy merge in my future...